docs/errors/usage_errors.md

```diff
@@ -1089,4 +1089,18 @@ pydantic.errors.PydanticUserError: Dataclass field bar has init=False and init_v
 """
 ```
 
+## `model_config` is used as a model property {#model-config-cannot-be-a-property}
```

Comment:
Below I suggested that we use `model-config-invalid-field-name`. Could you please change these docs to reflect that as well? I think we should use the term "model field" rather than property, just to keep things consistent 👍

Reply:
Also done